src/components/Review/ReviewSection.tsx

```diff
@@ -104,27 +105,53 @@ const ReviewSection: React.FC<ReviewSectionProps> = ({
+                              <Container textAlign="right">
+                                {review?.decision === undefined && (
+                                  <Button
+                                    size="small"
+                                    onClick={() => setDecisionArea(review, summaryViewProps)}
+                                    content={strings.BUTTON_REVIEW_RESPONSE}
+                                  />
+                                )}
+                              </Container>
+                            )}
+                          </Grid.Column>
+                        </Grid.Row>
+                        {review && review.decision && (
                           <Grid.Row>
-                            <Grid.Column>
-                              <SummaryViewWrapper {...summaryViewProps} />
-                            </Grid.Column>
-                            <Grid.Column>
-                              {review && canEdit && (
-                                <Container textAlign="right">
-                                  {review?.decision === undefined && (
-                                    <Button
-                                      size="small"
-                                      onClick={() => setDecisionArea(review, summaryViewProps)}
-                                      content={strings.BUTTON_REVIEW_RESPONSE}
-                                    />
-                                  )}
-                                </Container>
-                              )}
-                            </Grid.Column>
+                            <Card fluid>
+                              <Card.Content>
+                                <Grid>
+                                  <Grid.Row>
+                                    <Grid.Column width="10">
+                                      <Card.Header>{review.decision}</Card.Header>
+                                      <Card.Description>{review.comment}</Card.Description>
+                                      {assigned && (
+                                        <Card.Meta>{`${assigned.firstName} ${assigned.lastName}`}</Card.Meta>
+                                      )}
+                                    </Grid.Column>
+                                    <Grid.Column width="2">
+                                      <Icon
+                                        name="pencil square"
+                                        color="blue"
+                                        style={{ minWidth: 100 }}
+                                        onClick={() => setDecisionArea(review, summaryViewProps)}
+                                      />
+                                    </Grid.Column>
+                                  </Grid.Row>
+                                </Grid>
+                              </Card.Content>
+                            </Card>
                           </Grid.Row>
```

Comment:
Not sure why is showing all this diff here. Perhaps there is another PR included in this one too?

Reply:
There is actually a compilation error happening due to this change :(

Reply:
I think this is a conflict resolution mistake. It needs to be removed before merging!

Reply:
Weird. The diff I can explain, not so much the compilation error. The reason for the diff is that this whole block used to be within `if (category === TemplateElementCategory.Question) {`, but I removed that as we now show all, not just question types, hence the indentation for the whole block changed.

Have fixed the error now. If you look at the changes with "Hide whitespace changes" selected, you'll see the diff is just the couple of lines at each of the block.